source/collection-management/timeseries-collection.yml

```diff
@@ -0,0 +1,63 @@
+description: "timeseries-collection"
+
+schemaVersion: "1.0"
+
+runOnRequirements:
+  - minServerVersion: "4.9"
+    serverParameters:
+      featureFlagTimeseriesCollection:
+        value: true
+        version: "5.0"
```

Comment:
I realize the schema only defines [`serverParameters`](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/schema-1.1.json#L58) as an object type, but this seems very different from existing uses of `serverParameter` in the Versioned API tests, where all keys in the object have a scalar value.

Does `featureFlagTimeseriesCollection` actually have an object as its value? I would expect it to be a boolean like other feature flags.

[`runOnRequirement.serverParameter`](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#runonrequirement) doesn't say much about _how_ to match parameter values other than using flexible numeric comparisons. If some parameters can have non-scalar values I think we need to revise that text (and maybe suggest using document comparison rules defined elsewhere in the spec) -- and if so I'd suggest creating a ticket to track that work. cc: @alcaeus

Reply:
I had originally wanted to use the document matchers used elsewhere in the unified test format, but there were concerns around that. I believe equality matches on non-scalar values should still work, but we won't be able to use any matchers here.

Reply:
No objection on equality matches (and not supporting matching syntax), but we'd do well to use similar language to the `outcome` assertions:

> When comparing collection data, the rules in Evaluating Matches do not apply and the documents MUST match exactly; however, test runners MUST permit variations in document key order or otherwise normalize the documents before comparison.

yada yada JSON objects are unordered...

Reply:
Again, we no longer need to check this server parameter, but it would be good to update the spec language to clarify behavior with server parameters that are not just `true` or `false`.

Reply:
Will do by [DRIVERS-1765](https://jira.mongodb.org/browse/DRIVERS-1765).